contracts/MockContract.sol

```diff
@@ -288,7 +288,7 @@ contract MockContract is MockInterface {
 		// Clear list
 		methodIdMocks[SENTINEL_ANY_MOCKS] = SENTINEL_ANY_MOCKS;
 
-		fallbackExpectation = "";
+		fallbackExpectation = abi.encode(false);
```

Comment:
We could probably make this a constant `DEFAULT_FALLBACK_VALUE` or something ... but not super important for this contract